migrations/versions/f5babb123eb_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: f5babb123eb
+Revises: a22937d2373
+Create Date: 2016-03-21 14:17:11.725063
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'f5babb123eb'
+down_revision = 'a22937d2373'
+
+import fm  # noqa
+import sqlalchemy as sa  # noqa
+
+from alembic import op  # noqa
+from fm.ext import db  # noqa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('track_artist',
+    sa.Column('created', fm.db.types.UTCDateTime(), nullable=False),
+    sa.Column('updated', fm.db.types.UTCDateTime(), nullable=False),
+    sa.Column('artist_id', postgresql.UUID(), nullable=False),
+    sa.Column('track_id', postgresql.UUID(), nullable=False),
+    sa.ForeignKeyConstraint(['artist_id'], ['artist.id'], ),
+    sa.ForeignKeyConstraint(['track_id'], ['track.id'], ),
+    sa.PrimaryKeyConstraint('artist_id', 'track_id'),
+    sa.UniqueConstraint('artist_id', 'track_id')
+    )
+    op.create_index(op.f('ix_track_artist_artist_id'), 'track_artist', ['artist_id'], unique=False)
+    op.create_index(op.f('ix_track_artist_created'), 'track_artist', ['created'], unique=False)
+    op.create_index(op.f('ix_track_artist_track_id'), 'track_artist', ['track_id'], unique=False)
+    op.create_index(op.f('ix_track_artist_updated'), 'track_artist', ['updated'], unique=False)
+    ### end Alembic commands ###
+
+
+def downgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index(op.f('ix_track_artist_updated'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_track_id'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_created'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_artist_id'), table_name='track_artist')
+    op.drop_table('track_artist')
```

Comment:
Before we do any table dropping maybe we could have a little migration script that goes over our current list of album artists and tracks and fills this table?

Or could call the Spotify API to get the track artists on it's own.

Just don't wanna loose any data?